patches/openwrt/0027-ath79-enable-uart-for-Joy-IT-or750i.patch

```diff
@@ -0,0 +1,19 @@
+From: Jan-Niklas Burfeind <git@aiyionpri.me>
+Date: Wed, 5 Apr 2023 22:53:59 +0200
+Subject: ath79: enable uart for Joy-IT or750i
+
```

Comment:
Please add a short description of the problem to the patch file to have at hand when re-ordering the patches.